### PR TITLE
Fixed regression issue where setting tags for a redshift table was broken.

### PR DESCRIPTION
**Pull request by ajoymajumdar**
ConnectorBaseService.exists method was not implemented by the Redshift connector.
Fixed regression issue where getTable for non-existing redshift tables were not failing with a 404 error.

### CODE REVIEW

File: metacat-connector-jdbc/src/main/java/com/netflix/metacat/connector/jdbc/services/JdbcConnectorTableService.java [hunk 1]
```diff
@@ -94,15 +95,9 @@ public void delete(@Nonnull final ConnectorContext context, @Nonnull final Quali
...
-        try (final Connection connection = this.dataSource.getConnection()) {
-            final boolean upperCase = connection.getMetaData().storesUpperCaseIdentifiers();
-            if (upperCase) {
-                connection.setSchema(databaseName.toUpperCase(Locale.ENGLISH));
-            } else {
-                connection.setSchema(databaseName);
-            }
-            final String finalTableName = upperCase ? tableName.toUpperCase(Locale.ENGLISH) : tableName;
-            JdbcConnectorUtils.executeUpdate(connection, this.getDropTableSql(name, finalTableName));
+        try (Connection connection = this.dataSource.getConnection()) {
```
Comment (tgianos) [inline metacat-connector-jdbc/src/main/java/com/netflix/metacat/connector/jdbc/services/JdbcConnectorTableService.java]: Why did you remove the `final` from these try blocks?

File: metacat-connector-jdbc/src/main/java/com/netflix/metacat/connector/jdbc/services/JdbcConnectorTableService.java [hunk 2]
```diff
@@ -262,6 +253,23 @@ public void rename(
         }
     }
 
+    @Override
+    public boolean exists(@Nonnull final ConnectorContext context, @Nonnull final QualifiedName name) {
+        boolean result = false;
```
Comment (tgianos) [inline metacat-connector-jdbc/src/main/java/com/netflix/metacat/connector/jdbc/services/JdbcConnectorTableService.java]: this is the default value of a boolean anyway you don't need to initialized it.

File: metacat-connector-jdbc/src/main/java/com/netflix/metacat/connector/jdbc/services/JdbcConnectorTableService.java [hunk 3]
```diff
@@ -262,6 +253,23 @@ public void rename(
...
+        try (Connection connection = this.dataSource.getConnection()) {
+            final String databaseName = name.getDatabaseName();
+            connection.setSchema(databaseName);
+            final DatabaseMetaData metaData = connection.getMetaData();
+            final ResultSet rs = metaData.getTables(databaseName, databaseName, name.getTableName(), TABLE_TYPE);
+            if (rs.next()) {
```
Comment (tgianos) [inline metacat-connector-jdbc/src/main/java/com/netflix/metacat/connector/jdbc/services/JdbcConnectorTableService.java]: here it might be better to do `rs.last()` and then `return rs.row() == 1`. That way we verify that the table exists and it's the only one.

File: metacat-connector-jdbc/src/main/java/com/netflix/metacat/connector/jdbc/services/JdbcConnectorTableService.java (hunk 1 above)
Comment (ajoymajumdar) [inline metacat-connector-jdbc/src/main/java/com/netflix/metacat/connector/jdbc/services/JdbcConnectorTableService.java]: Not needed in try-with-resources statement.

File: metacat-connector-jdbc/src/main/java/com/netflix/metacat/connector/jdbc/services/JdbcConnectorTableService.java (hunk 2 above)
Comment (ajoymajumdar) [inline metacat-connector-jdbc/src/main/java/com/netflix/metacat/connector/jdbc/services/JdbcConnectorTableService.java]: At a method level, you need to initialize.

File: metacat-connector-jdbc/src/main/java/com/netflix/metacat/connector/jdbc/services/JdbcConnectorTableService.java (hunk 3 above)
Comment (ajoymajumdar) [inline metacat-connector-jdbc/src/main/java/com/netflix/metacat/connector/jdbc/services/JdbcConnectorTableService.java]: Using next() should be acceptable with the assumption that databases will not have two tables with the same name.
